docs/NooBaaNonContainerized/NooBaaCLI.md

```diff
@@ -458,8 +462,66 @@ noobaa-cli diagnose metrics
 The `gather-logs` command is used for extract NooBaa non containerized logs.
 Not implemented yet, running this command will fail with not implemented error.
 
+## Upgrade
 
-## Global Flags
+The `upgrade` command is being used for running config directory upgrade operations.
+- **[Start](#upgrade-start)**: Initiate config directory upgrade.
+- **[Status](#upgrade-status)**: Retrieve the in progress config directory upgrade status.
+- **[History](#upgrade-history)**: Retrieve the history information of past config directory upgrades.
+
+For more information about the config directory upgrade, See - [Upgrade](./Upgrade.md#online-upgrade-version--5180)
+
+### Upgrade Start
+
+The `upgrade start` command is used to start a config directory upgrade run.
+
+#### Usage
+```sh
+noobaa-cli upgrade start --expected_version --expected_hosts [--skip-verification] [--custom_upgrade_scripts_dir]
+```
+
+#### Flags -
+- `expected_version` (Required)
+    - Type: String
+    - Description: Specifies the upgrade's expected target version.
+    - Example - `--expected_version 5.18.0`
```

Comment:
Do we need to wrap it with quotes? (`--expected_version '5.18.0'`)

Reply:
no